manim/mobject/types/vectorized_mobject.py

```diff
@@ -67,7 +68,7 @@ def __new__(cls, name, bases, namespace):
         return super().__new__(cls, name, bases, namespace)
 
 
-class VMobject(Mobject):
+class VMobject(Mobject, metaclass=ConvertToOpenGL):
```

Comment:
I think we should -- for now -- keep `VMobject` and `Mobject` as-is: I don't think that this OpenGL-ification of `VMobject` would work given how different the implementation of `OpenGLVMobject` is.